yt_dlp/extractor/pixivsketch.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    ExtractorError,
+    unified_timestamp,
+)
+
+
+class PixivSketchBaseIE(InfoExtractor):
+    def _call_api(self, video_id, path, referer):
+        response = self._download_json(f'https://sketch.pixiv.net/api/{path}', video_id, headers={
+            'Referer': referer,
+            # this is correct
```

Comment:
comment is not needed

Reply:
really? the value for `X-Requested-With` is non-standard here

Reply:
yeah
It's assumed that all extractors are gone through robust review process and all the params are the essential ones in web requests....so if a param is there, it must be for a reason no matter how absurd it looks/feels (or atleast that's how i see it)

Reply:
okay, will remove